stan/math/prim/scal/fun/log1m.hpp

```diff
@@ -33,10 +34,13 @@ namespace math {
  *
  * @param[in] x Argument.
  * @return Natural log of one minus the argument.
- * @throw std::domain_error If the argument is greater than 1.
- * @throw std::overflow_error If the computation overflows.
+ * @throw <code>std::domain_error</code> If the argument is greater than 1.
+ * @throw <code>std::overflow_error</code> If the computation overflows.
  */
-inline double log1m(double x) { return stan::math::log1p(-x); }
+inline double log1m(double x) {
+  check_less_or_equal("log1m(x)", "x", x, 1);
```

Comment:
replace `"log1m(x)"` with `"log1m". You'll need to update the test to match. (The rest of our error messages just have the function name followed by colon, no arguments in the function)